source/collectBy.js

```diff
@@ -0,0 +1,37 @@
+import _curry2 from './internal/_curry2';
+import groupBy from './groupBy';
+import values from './values';
+
+/**
+ * Splits a list into sub-lists, based on the result of calling a String-returning function on each element,
+ * and grouping the results according to values returned.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig (a -> String) -> [a] -> {String: [a]}
+ * @param {Function} fn Function :: a -> String
+ * @param {Array} list The array to split into sub-arrays.
+ * @return {Array}
+ *    An array of arrays where each sub-array contains items for which
+ *    the String-returning function has returned the same value.
+ * @see R.groupBy
```

Comment:
Should `R.partition` be mentioned?

Reply:
Yes, I think that's a good idea.  It's quite related.  In fact, `collectBy(pred)` will group a list into the same groups that `partition(pred)` does, although the order might not be the same.

Reply:
Mentioned `partition` in see tag